tests/Microsoft.Identity.Test.Unit.net45/TestBase.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Test.Common;
+using Microsoft.Identity.Test.Common.Core.Mocks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Unit
+{
+    [TestClass]
+    public class TestBase
```

Comment:
Having a TestBase is both a blessing and curse. MSTest is exceptionally bad at mixing attributes and inheritance...

Anyway, test classes inheriting from this will benefit from TestInitialize. CreateTestHarness has added benefit that it can inject the TestContext in test code deeper down, e.g. in MockHttpManager. TestContext has properties such as TestName, which is useful for logging and tracing.